package.json

```diff
@@ -10,6 +10,7 @@
     "preview": "vite preview"
   },
   "dependencies": {
+    "@unocss/reset": "^0.55.7",
```

Comment:
`@unocss/reset` [está incluido](https://github.com/unocss/unocss/blob/main/packages/README.md) por defecto en `unocss`

Reply:
seguro? porque me daba errores ...
![image](https://github.com/Afordin/hackafor/assets/56722632/228f48b0-39e6-4bf3-a6e6-1ff91b03193d)

Reply:
mmm no si la borro aun me da problemas 
![image](https://github.com/Afordin/hackafor/assets/56722632/a62dec61-5e42-4213-b26c-e51dc4c7aacf)

Reply:
una vez agrego `@unocss/reset` ya me anda
![image](https://github.com/Afordin/hackafor/assets/56722632/eab935af-3964-43f2-accd-7af147a28bc0)

Reply:
El problema es por el hoisting de PNPM. `@unocss/reset` es una [dependencia fantasma](https://www.coana.tech/post/a-quick-introduction-to-phantom-dependencies) de `unocss`. Si bien esto no da problemas en otros gestores como Yarn o NPM, en PNPM tienes que añadir lo siguiente a `.npmrc` (en la raíz del proyecto):
```yml
public-hoist-pattern[]=*unocss*
```
Y reinstalar las dependencias

Reply:
Grande owlnai! https://github.com/Afordin/hackafor/pull/2/commits/8e08ec7c389d7c9a994fdc0870fd884193eaa9ab

Reply:
Resolvidito?

Reply:
Sí, gracias! :)